test/scripts/region_loading-01.js

```diff
@@ -0,0 +1,38 @@
+// Test basic logpoint functionality. When logpoints are added,
+Test.describe(
+  `new messages should appear in the correct order and allow time warping.`,
+  async () => {
+    const { assert } = Test;
+
+    await Test.selectSource("doc_rr_basic.html");
+    await Test.addBreakpoint("doc_rr_basic.html", 20);
+    await Test.setBreakpointOptions("doc_rr_basic.html", 20, undefined, {
+      logValue: `"Logpoint Number " + number`,
+    });
+    await Test.addBreakpoint("doc_rr_basic.html", 9, undefined, {
+      logValue: `"Logpoint Beginning"`,
+    });
+    await Test.addBreakpoint("doc_rr_basic.html", 7, undefined, {
+      logValue: `"Logpoint Ending"`,
+    });
+
+    await Test.selectConsole();
+
+    const messages = await Test.waitForMessageCount("Logpoint", 12);
+    assert(!Test.findMessages("Loading").length);
+
+    assert(messages[0].textContent.includes("Beginning"));
+    for (let i = 1; i <= 10; i++) {
+      assert(messages[i].textContent.includes("Number " + i));
+    }
+    assert(messages[11].textContent.includes("Ending"));
+
+    await Test.warpToMessage("Number 5");
+
+    await Test.checkEvaluateInTopFrame("number", 5);
+    await Test.reverseStepOverToLine(19);
+
+    // The logpoint acts like a breakpoint when resuming.
+    await Test.resumeToLine(20);
+  }
+);
```

Comment:
Test looks good, but it's not clear how this is testing region loading?

Reply:
It's not.  It's just a copy of the test files.  I didn't add reviewers to this yet, but mostly wanted to see if I could make an new test file and have it pass.